requirements.txt

```diff
@@ -0,0 +1,8 @@
+arrow
+bleach
+fedmsg[all]
```

Comment:
No it's good. Sometimes I make silly changes that made sense at the time, or were tests, and they slip into my PRs :smile: - being able to justify my changes makes me feel comfortable.